pkg/api/types.go

```diff
@@ -547,6 +547,27 @@ type PersistentVolumeClaimSpec struct {
 	StorageClassName *string
 }
 
+type PersistentVolumeClaimConditionType string
+
+// These are valid conditions of Pvc
+const (
+	// An user trigger resize of pvc has been started
+	PersistentVolumeClaimResizeStarted PersistentVolumeClaimConditionType = "ResizeStarted"
```

Comment:
This can be a follow up but should be "Resizing", not "ResizingStarted".  A condition is level driven, not edge driven

Reply:
You mean both Constant and its value should be - `PersistentVolumeClaimResizing` and `Resizing` ?

Reply:
done.